test/mounted_files_test.py

```diff
@@ -123,14 +123,19 @@ def test_mounted_files_package_no_automount(supports_dir, env_mount_files, servi
     p = subprocess.run(
         ["modal", "run", "pkg_a.package"],
         cwd=supports_dir,
-        capture_output=True,
         env={**os.environ, "MODAL_AUTOMOUNT": "0"},
     )
     assert p.returncode == 0
     files = set(servicer.files_name2sha.keys()) - set(env_mount_files)
     assert files == {
         "/root/pkg_a/__init__.py",
+        "/root/pkg_a/a.py",
```

Comment:
Change to the automount=0 behavior results in this, as documented by the second changelog bullet